ee/modules/110-istio/hooks/revisions_monitoring.go

```diff
@@ -169,6 +169,15 @@ func revisionsMonitoring(input *go_hook.HookInput, dc dependency.Container) erro
 			}
 		}
 
+		// TODO: get rid of everything but this only metric https://github.com/deckhouse/deckhouse/issues/2182
+		labels := map[string]string{
+			"namespace":        istioPod.GetNamespace(),
+			"dataplane_pod":    istioPod.GetName(),
+			"desired_revision": desiredRevision,
```

Comment:
Probably "pod" will do? Why do wee need this "dataplane" detail?

Reply:
Deckhouse will rewrite it to exported_pod. The "pod" will be smth like "deckhouse-xxx-yyy".

Reply:
Oh, I see. Thanks